sfputil/main.py

```diff
@@ -1112,6 +1112,61 @@ def run_firmware(port_name, mode):
 
     return status
 
+def is_fw_switch_done(port_name):
```

Comment:
I am not sure why this API is required because there is already a return status from run_firmware() to indicate a failure. An imageA could be running while the committed Image could be Image B. For eg.

1. Image A is running and its is the committed image
2. User updates Image B
3. Runs the image B

Now the running image is B but the committed image is A

Reply:
"run_firmware" indicates the status which read before reset to target image. This status only means module received and done the judgement for "run_firmware" cmd, but not means module already jumped into target image.
"run_firmware" will maybe involved non-volatile memory operation in module according to module implementation, so that it will maybe spend several or tens of seconds. That will cause commit failed if sends "commit_firmware" once "run_firmware" done without "is_fw_switch_done".
![imageCheckPurpose](https://user-images.githubusercontent.com/28720623/193781661-1318b845-b10e-4967-9bb0-8c04ddb08063.png)